src/components/layout/header/platform-switcher/index.tsx

```diff
@@ -1,13 +1,75 @@
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { DropdownMenu, DropdownMenuTrigger, DropdownMenuContent } from '../../../ui/dropdown-menu/index';
+import PlatformSwitcherCard from './platform-switcher-card';
+
+export interface TAvailableApp {
+    src: string;
+    title: string;
+    alt: string;
+    path: string;
+}
 
 const PlatformSwitcher = () => {
     const { t } = useTranslation();
+
+    const [isDropdownOpen, setDropdownOpen] = useState<boolean>(false);
+
+    const handleDropdownToggle = () => {
+        setDropdownOpen(!isDropdownOpen);
+    };
+
+    const available_apps: TAvailableApp[] = [
+        {
+            src: '/images/common/ic-rebranding-deriv-trader.svg',
+            title: 'A whole new trading experience on a powerful yet easy to use platform.',
+            alt: 'dtrader icon',
+            path: '/',
+        },
+        {
+            src: '/images/common/ic-rebranding-deriv-bot.svg',
+            title: 'Automated trading at your fingertips. No coding needed.',
+            alt: 'dbot icon',
+            path: '/',
+        },
+        {
+            src: '/ic_smart_trader.svg',
+            title: 'Trade the world’s markets with our popular user-friendly platform.',
+            alt: 'dsmart trader icon',
+            path: '/',
+        },
+        {
+            src: '/images/common/ic-rebranding-binary-bot.svg',
+            title: 'Our classic “drag-and-drop” tool for creating trading bots, featuring pop-up trading charts, for advanced users.',
+            alt: 'binary bot icon',
+            path: '/',
+        },
+    ];
```

Comment:
Keeping this outside component is better approach. In this way, It will re-run every time the component is re-rendered